testsuite/features/support/constants.rb

```diff
@@ -1509,7 +1508,7 @@
   'sle-manager-tools-for-micro5-updates-x86_64-5.4' => 60,
   'sle-manager-tools-for-micro5-updates-x86_64-5.5' => 60,
   'sle-manager-tools-for-micro6-updates-x86_64-6.0' => 60,
-  'sle-micro-5.3-devel-uyuni-client-x86_64' => 60,
+  # TODO Enable when it becomes available 'sle-micro-5.3-devel-uyuni-client-x86_64' => 60,
```

Comment:
That was the wrong channel to comment out :wink:

Reply:
fixed again, thanks for catching all these, I am more tired than I thought.

Reply:
No worries and thank you for fixing!